Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+
+namespace BootCamp.Chapter
+{    public class Lesson3
+    {   public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Clear();
+                GatherData(out string firstName, out string lastName, out int age, out float weight, out float height);
+                DisplayOutput(firstName, lastName, age, weight, height);
+            }
+        }
+        public static string GetStringInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            return Console.ReadLine();
+        }
+        public static int GetIntInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            return Convert.ToInt32(Console.ReadLine());
+        }
+        public static float GetFloatInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            return float.Parse(Console.ReadLine());
+        }
+        // Input function
+        public static void GatherData(out string firstName, out string lastName, out int age, out float weight, out float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider refactoring this method in order to remove the need for this 'out' modifier.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845599)